Packs/CortexAttackSurfaceManagement/Scripts/InferWhetherServiceIsDev/InferWhetherServiceIsDev_test.py

```diff
@@ -72,7 +73,7 @@ def test_get_indicators_from_external_classification(classifications, matches):
 def test_determine_reason(external, internal, reason):
     from InferWhetherServiceIsDev import determine_reason
 
-    assert determine_reason(external, internal) == reason
+    assert determine_reason(external, internal, [], "") == reason
```

Comment:
Do we want another test for `if provider:` in `determine_reason` or is provider not functionally relevant?

Reply:
I am not doing full testing on determine_reason(). i can, but next person will have to maintain 93%+ unit test coverage